modules/cluster-resizing/pages/expand-a-cluster.adoc

```diff
@@ -17,6 +17,21 @@ WARNING: If your TigerGraph cluster uses a seeded license, please https://tigerg
 * Obtain a few key measures for the state of your data before the expansion, such as vertex counts/edge counts or certain query results. This will be useful in verifying data integrity after the expansion completes.
 * If the original cluster is a single node installation, make sure the IP used is not a https://en.wikipedia.org/wiki/Localhost[local loopback address] such as 127.0.0.1.
 
+[WARNING]
+====
+If you ever deleted a vertex attribute, cluster expansion might lead to data corruption.
```

Comment:
Maybe "if you have ever deleted a vertex attribute..." 
Do we need to say "This is a known issue"?